topic5/src/main/java/highSchool/topic5/Course.java

```diff
@@ -0,0 +1,185 @@
+package highSchool.topic5;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+public class Course {

```

Comment:
It's not clear which is the responsibility of this object, if I look at the name I would expect it to model a Course. Also method names do not denote object behavior. I think what you have here is a "CoursePrinterService" with operations like printInfoForCourse(int courseId) throws SQLException, printPercentagePassFail..etc